src/styles/less/AccountSettings.less

```diff
@@ -48,7 +48,9 @@
         background-color: #8ce2e0;
     }
 }
-
+.settings-buttons-cotainer {
+  border: 1px solid red;
+}
   
```

Comment:
I imagine this was just for dev purposes and will be removed at a later time?

Reply:
Yes, it will be removed when I add the proper styling. it's not visible on the page.